### ISSUE TITLE
Using Training data with SHAP

### ISSUE THREAD

**Issue by qdickinson** [shap/shap#2055]
Hello,

I am looking to analyze SHAP values to better understand the models I have trained. The issues I am running into is the sample size of the test dataset. While I get good explanations for the test set, I cannot make statistically significant claims. In order to increase the statistical power of my analysis, I thought I may be able to use the training set to get model explanations. But I am not sure if there is something potentially wrong with this approach. From the data I have analyzed so far, the patterns seem consistent between the test and the training data. However, all uses of SHAP I have seen only use the test data to get explanations. Is it acceptable to use training data in this manner?

I saw a similar question in a previous [issue](https://github.com/slundberg/shap/issues/833) as well.

Thank you.
